src/main/java/mServer/crawler/sender/MediathekSrf.java

```diff
@@ -260,6 +273,18 @@ private void addFilms(String jsonMovieUrl, String themePageUrl, String theme) {
             }
         }
 
+        private void getM3u8(String url3u8) {
+            m3u8Page = getUrl.getUri_Utf(SENDERNAME, url3u8, m3u8Page, "");
+            if (m3u8Page.length() == 0) {
+                // tauschen https://srfvodhd-vh.akamaihd.net http://hdvodsrforigin-f.akamaihd.net
+                // ist ein 403
+                if (url3u8.startsWith("https://srfvodhd-vh.akamaihd.net")) {
+                    url3u8 = url3u8.replaceFirst("https://srfvodhd-vh.akamaihd.net", "http://hdvodsrforigin-f.akamaihd.net");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "url3u8". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)